### PR TITLE
[PSR-18] Added section about why we named it "sendRequest"

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: proposed/http-client/http-client-meta.md [hunk 1]
```diff
@@ -82,6 +82,15 @@ and second by catching potential exceptions.
 To make the specification more predictable, it was decided that HTTP clients never will throw
 exceptions for 4xx and 5xx responses.
 
+#### Why "sendRequest"?
+
+The method signature `ClientInterface::sendRequst(RequstInterface $request): ResponseInterface`
```
Comment (dazz) [inline proposed/http-client/http-client-meta.md]: `sendRequst` => `sendRequest`

File: proposed/http-client/http-client-meta.md [hunk 2]
```diff
@@ -82,6 +82,15 @@ and second by catching potential exceptions.
...
+could be considered to be a bit too verbose. An initial idea would be to name it
+`ClientInterface::send(RequstInterface $request): ResponseInterface`. That would
+cause issues when major HTTP client libraries will implement the `ClientInterface` because
+most of them already have defined a function `send`. To allow those libraries implement
+`ClientInterface` without breaking backward compatibility we simply renamed it `sendReqeust`.
```
Comment (dazz) [inline proposed/http-client/http-client-meta.md]: `sendReqeust` => `sendRequest`

File: proposed/http-client/http-client-meta.md (hunk 1 above)
Comment (dazz) [reply proposed/http-client/http-client-meta.md]: and `RequstInterface` => `RequestInterface`

File: proposed/http-client/http-client-meta.md [hunk 3]
```diff
@@ -82,6 +82,15 @@ and second by catching potential exceptions.
 To make the specification more predictable, it was decided that HTTP clients never will throw
 exceptions for 4xx and 5xx responses.
 
+#### Why "sendRequest"?
+
+The method signature `ClientInterface::sendRequst(RequstInterface $request): ResponseInterface`
+could be considered to be a bit too verbose. An initial idea would be to name it
+`ClientInterface::send(RequstInterface $request): ResponseInterface`. That would
```
Comment (dazz) [inline proposed/http-client/http-client-meta.md]: sorry, even one more :) `RequstInterface` => `RequestInterface`

**Comment by dazz** [comment]
I found some Rechtschreibfehler. Please corect. Thanks!

**Comment by Nyholm** [comment]
Hey. Sorry, did I spell it correct even once? 

Thank you for correcting me.

File: proposed/http-client/http-client-meta.md [hunk 4]
```diff
@@ -82,6 +82,15 @@ and second by catching potential exceptions.
 To make the specification more predictable, it was decided that HTTP clients never will throw
 exceptions for 4xx and 5xx responses.
 
+#### Why "sendRequest"?
```
Comment (sagikazarmark) [inline proposed/http-client/http-client-meta.md]: I rephrased it a bit, hope you don't mind.

```
The method signature `ClientInterface::sendRequest(RequestInterface $request): ResponseInterface`
might sound a bit verbose at first. An obvious improvement would be to just
call the method `send`.

The reason behind this verbosity is how existing HTTP client libraries are implemented.
Most of them already have a method called `send` with arbitrary signature.
To make adoption easier we decided to choose a more verbose name.
Besides the backward compatibility argument, future HTTP client implementations
will also have the ability to implement an arbitrary API using the name `send`
and still implement this standard. For example a simplified method signature like
`send(string $method, string $url, array $headers, string $body): ResponseInterface`
can be a common use case.
```

Comment (Nyholm) [reply proposed/http-client/http-client-meta.md]: Thank you

File: proposed/http-client/http-client-meta.md [hunk 5]
```diff
@@ -82,6 +82,21 @@ and second by catching potential exceptions.
...
+
+The method signature `ClientInterface::sendRequest(RequestInterface $request): ResponseInterface`
+might sound a bit verbose at first. An obvious improvement would be to just
+call the method `send`.
+
+The reason behind this verbosity is how existing HTTP client libraries are implemented.
+Most of them already have a method called `send` with an arbitrary signature.
+To make adoption easier we decided to choose a more verbose name.
+Besides the backward compatibility argument, future HTTP client implementations
```
Comment (dbu) [inline proposed/http-client/http-client-meta.md]: i would drop this second part. if we called our thing `send`, the future clients could call their convenience method `get` and `post` etc, for example.

File: proposed/http-client/http-client-meta.md [hunk 6]
```diff
@@ -82,6 +82,21 @@ and second by catching potential exceptions.
...
 
+#### Why "sendRequest"?
+
+The method signature `ClientInterface::sendRequest(RequestInterface $request): ResponseInterface`
+might sound a bit verbose at first. An obvious improvement would be to just
+call the method `send`.
+
+The reason behind this verbosity is how existing HTTP client libraries are implemented.
+Most of them already have a method called `send` with an arbitrary signature.
+To make adoption easier we decided to choose a more verbose name.
```
Comment (dbu) [inline proposed/http-client/http-client-meta.md]: missing comma: "adoption easier, we decided"

**Pull request closed without merging**